tests/dashboard/service_test.go

```diff
@@ -156,21 +144,6 @@ func (s *serverTestSuite) TestDashboard(c *C) {
 	s.checkServiceIsStarted(c, servers, leader)
 	c.Assert(leader.GetServer().GetScheduleOption().GetDashboardAddress(), Equals, dashboardAddress2)
 
-	// Changing dashboard address
```

Comment:
Do we have other ways to run this test?

Reply:
Any suggestion?

Reply:
I will find a way to make up for this test later.